scorecards-site/static/viewer/index.html

```diff
@@ -783,6 +795,23 @@
       </tr>
     </script>
 
+    <!-- Search template -->
+    <script id="search-template" type="text/x-handlebars-template">
+      <tr>
+        <td class="{{#if alignment}}text-{{alignment}}{{/if}}" colspan="2"">
+          <form action="" method="get">
+            <div class="table-cell form-content">
+              <label for="uri">Repo in <code>domain/org/repo</code> format: </label>
```

Comment:
Let's call it `platform` for consistency elsewhere.

Reply:
Done.